game-core/src/main/java/org/triplea/lobby/common/GameDescription.java

```diff
@@ -0,0 +1,68 @@
+package org.triplea.lobby.common;
+
+import java.io.Serializable;
+import java.time.Instant;
+
+import org.triplea.game.server.HeadlessGameServer;
+
+import games.strategy.net.INode;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import lombok.Value;
+import lombok.experimental.Wither;
+
+/**
+ * Immutable Data class being used to send information about the
+ * current game state to the lobby.
+ */
+// See https://github.com/google/error-prone/pull/1195 and https://github.com/rzwitserloot/lombok/issues/737
+@SuppressWarnings("ReferenceEquality")
+@Value
+@Builder
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@EqualsAndHashCode
+@ToString
+public class GameDescription implements Serializable {
+  private static final long serialVersionUID = 508593169141567546L;
+
+  /**
+   * Represents the game states displayed to users looking at the list of available lobby games.
+   */
+  @AllArgsConstructor
+  public enum GameStatus {
+    LAUNCHING("Launching"), IN_PROGRESS("In Progress"), WAITING_FOR_PLAYERS("Waiting For Players");
```

Comment:
Spotless wants it like that

Reply:
FYI: two hacks for that:
(1) double spacing, eg:
```
LAUNCHING(..),

IN_PROGRESS(..)
:
:
```

(2) IMO v. ugly, end of line empty comment:
```
LAUNCHING(..), //
IN_PROGRESS(..) //
:
```

Google-java-format FWIW will do one enum per-line. Until then, if desired, I'd recommend hack (1). Not a huge deal as well because the enum is short.